api/v1/canary_types.go

```diff
@@ -197,6 +198,9 @@ func (spec CanarySpec) GetAllChecks() []external.Check {
 	for _, check := range spec.Opensearch {
 		checks = append(checks, check)
 	}
+	for _, check := range spec.Webhook {
```

Comment:
Its probably not necessary to include this in GetAllChecks

Reply:
I had it excluded initially but the check was never saved by the `SyncCanaryJob` because `RunChecks` returned 0 results.